pylint_odoo/checkers/no_modules.py

```diff
@@ -243,6 +248,11 @@
     '__init__', 'setUp', 'tearDown',
 ]
 
+DFTL_DEPRECATED_FIELD_PARAMETERS = {
+    'digits_compute': 'digits',
+    'select': 'index',
+}
+
 
 class NoModuleChecker(BaseChecker):
```

Comment:
Use a options in order to add this values from pylintrc file.
Looking for similar examples with other `DFLT` variables used from options